src/globals/scss/_common.scss

```diff
@@ -0,0 +1,41 @@
+// This file is included by each individual component as well as the
+// govuk-frontend file itself.
+//
+// This means that every component has access to all things within the settings,
+// tools and helpers layers
+
+// Settings
+@import "settings/colours-palette";
+@import "settings/colours-organisations";
+@import "settings/colours-applied";
+@import "settings/spacing";
+@import "settings/measurements";
+@import "settings/font-stack";
+@import "settings/media-queries";
+@import "settings/paths";
+
+// Tools
+@import "tools/exports";
+@import "tools/file-url";
+@import "tools/legacy-ie";
+@import "tools/px-to-em";
+
+// Vendor libraries
+@import "./node_modules/sass-mq/mq"; // We use sass-mq module for media queries
```

Comment:
I have mixed feelings about this being here, but I don't see a better place right now.
I don't think the dependency on `sass-mq` is documented anywhere and it probably should.

Reply:
Yeah, thinking is that it's _essentially_ part of our tools layer, it's just rather than rolling our own we're including another library. Open to suggestions, but I think we can work around this with documentation.